Library/ViewModels/DiscoveryPostcardViewModelTests.swift

```diff
@@ -159,7 +183,7 @@ internal final class DiscoveryPostcardViewModelTests: TestCase {
         self.saveButtonSelected.assertValues([true], "Save button is selected at first.")
         self.saveButtonEnabled.assertValueCount(0)
 
-        self.vm.inputs.saveButtonTapped()
+        self.vm.inputs.saveButtonTapped(selected: true)
```

Comment:
If you want to omit these explicit parameters passed into the function you can make the default value `true`? So that you don't have to change the call site here and it could stay `self.vm.inputs.saveButtonTapped()`

so the function would be
```
func saveButtonTapped(selected: Bool = true) { ... }
```